packages/react-text/src/components/Text/useText.ts

```diff
@@ -12,17 +10,27 @@ const mergeProps = makeMergeProps<TextState>();
  *
  * @param props - props from this instance of Text
  * @param ref - reference to root HTMLElement of Text
- * @param defaultProps - (optional) default prop values provided by the implementing type
  */
-export const useText = (props: TextProps, ref: React.Ref<HTMLElement>, defaultProps?: TextProps): TextState => {
-  const state = mergeProps(
-    {
-      ref,
-      as: 'span',
-    },
-    defaultProps,
-    props,
-  );
+export const useText = (props: TextProps, ref: React.Ref<HTMLElement>): TextState => {
+  return {
+    wrap: true,
+    truncate: false,
+    block: false,
+    italic: false,
+    underline: false,
+    strikethrough: false,
+    size: 300,
+    font: 'base',
+    weight: 'regular',
+    align: 'start',
+
+    components: { root: 'span' },
 
-  return state;
+    ...props,
```

Comment:
I'm not sure if this needs to be here? I believe that `...props` just goes on the root slot.

Reply:
```suggestion
```

@czearing yes, it's correct. Overlooked it 🙄

Reply:
I seem to be missing something then. How will ` useTextStyles`  hook have access to the props then?

Reply:
From what I know, the props that are used for styling and what not are deconstructed and passed on:
```jsx
  const { value, defaultValue } = props;
  const state: ComponentState = {
    value,
    defaultValue,
    root: getNativeElementProps('div', {
      ...
    }),
```
Everything else gets passed to the root:
```jsx
    root: getNativeElementProps('span', {
      ref,
      ...props,
      id: useId('componentName-', props.id),
    }),
```